alpakka-sample-file-to-elasticsearch/project/Dependencies.scala

```diff
@@ -3,23 +3,23 @@ import sbt._
 object Dependencies {
   val scalaVer = "2.13.8"
   // #deps
-  val AkkaVersion = "2.6.19"
-  val AlpakkaVersion = "4.0.0"
+  val PekkoVersion = "1.0.0"
+  val PekkoConnectorsVersion = "0.0.0+129-1853d802-SNAPSHOT" // #TODO: Change to release version
   // #deps
   val dependencies = List(
     // #deps
-    "com.lightbend.akka" %% "akka-stream-alpakka-file" % AlpakkaVersion,
-    "com.lightbend.akka" %% "akka-stream-alpakka-elasticsearch" % AlpakkaVersion,
-    "com.typesafe.akka" %% "akka-stream" % AkkaVersion,
-    "com.typesafe.akka" %% "akka-actor-typed" % AkkaVersion,
-    "com.typesafe.akka" %% "akka-actor" % AkkaVersion,
+    "org.apache.pekko" %% "pekko-connectors-file" % PekkoConnectorsVersion,
+    "org.apache.pekko" %% "pekko-connectors-elasticsearch" % PekkoConnectorsVersion,
+    "org.apache.pekko" %% "pekko-stream" % PekkoVersion,
+    "org.apache.pekko" %% "pekko-actor-typed" % PekkoVersion,
+    "org.apache.pekko" %% "pekko-actor" % PekkoVersion,
     // for JSON in Scala
     "io.spray" %% "spray-json" % "1.3.6",
     // for JSON in Java
     "com.fasterxml.jackson.datatype" % "jackson-datatype-jdk8" % "2.13.4",
```

Comment:
Can you change jackson to 2.14.3? That's what Pekko 1.0.0 uses.

Reply:
Resolved: [e4cb00d](https://github.com/apache/incubator-pekko-connectors-samples/pull/2/commits/e4cb00df28f744aa72cdda6a55d4e151ade42ae6)